exonum_client/client.py

```diff
@@ -451,7 +454,7 @@ def get_main_proto_sources(self) -> List[ProtoFile]:
 
     def get_proto_sources_for_artifact(self, runtime_id: int, artifact_name: str) -> List[ProtoFile]:
```

Comment:
Nit: `runtime_id` parameter is not needed anymore?

Reply:
Or, maybe, it makes sense to throw an exception if `runtime_id` is not equal to the rust runtime ID.

Reply:
Raise an exception if `runtime_id != 0`?

Reply:
I guess it would be better to completely remove it.

Reply:
>Raise an exception if runtime_id != 0?

Yep, just so if user will try to load proto sources from other runtime, it will fail.

Reply:
Done